training/src/anemoi/training/train/forecaster/interpolator.py

```diff
@@ -99,8 +100,15 @@ def _step(
 
         # Delayed scalers need to be initialized after the pre-processors once
         if self.is_first_step:
-            self.define_delayed_scalers()
+            self.update_scalars(callback=AvailableCallbacks.ON_TRAINING_START)
             self.is_first_step = False
+        self.update_scalars(
+            callback=(
```

Comment:
I think we thought that the valid/training abstraction was not needed, and just keep the training start and batch one. So this would need update?